frontend/src/api/DeviceApi.js

```diff
@@ -97,3 +97,10 @@ export function getReceivers(callback) {
       SampleData.getReceivers(callback);
     });
 }
+
+export function deleteDevice(deviceId, callback) {
+  axios
+    .delete(`REACT_APP_DEVICE=$DOMAIN/device/${deviceId}`)
+    .then(callback)
+    .catch((error) => {});
```

Comment:
If you aren't doing anything with the error, I don't think you need to put in that argument in the lambda function

Reply:
Not related to my PR, so it would be better to address this issue in @gdzooks's #248 PR.

Reply:
ideally that's where we'd put error messages n stuff but you're right that it's doing nothing rn. i'll remove it from StreamApi as well since i did the same thing there